src/aws.py

```diff
@@ -1,35 +1,9 @@
 from buildercore import core
 import utils
 from decorators import requires_aws_stack, debugtask, echo_output
-from buildercore.core import MultipleRegionsError
-
 import logging
 LOG = logging.getLogger(__name__)
 
-def find_region(stackname=None):
-    """tries to find the region, but falls back to user input if there are multiple regions available.
-
-    Uses stackname, if provided, to filter the available regions"""
-    try:
-        return core.find_region(stackname)
-    except MultipleRegionsError as e:
-        print("many possible regions found!")
-        return utils._pick('region', e.regions())
-
-def stack_list(region=None):
```

Comment:
this method was dead code, good to remove it

Reply:
yeah, always a toss up between marking it as 'dead' or just outright deleting it.